README.md

```diff
@@ -56,7 +56,7 @@ The common case starts an `ActiveSpan` that's automatically registered for intra
 ```java
 io.opentracing.Tracer tracer = ...;
 ...
-try (ActiveSpan activeSpan = tracer.buildSpan("someWork").startActive()) {
+try (ActiveSpan activeSpan = tracer.buildSpan("someWork").startActive(new AutoFinisher())) {
```

Comment:
it seems the `try()` would still call `close()` and finish the span, whether the AutoFinisher is passed or not.